app/views/vacancies/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<h2 class="page-title"><%= t(".edit_vacancies") %></h2>
+
+
+<div class="panel panel-default">
```

Comment:
All inside this `div` is equal the `new.html.erb`, right?

You can move this to a partial called `_form.html.erb`, and here (and in the new.html.erb), call it by

```
<h2 class="page-title"><%= t(".edit_vacancies") %></h2>

<%= render 'form' %>
```